### DISCUSSION TITLE
El selector múltiple de tipo usuarios, al seleccionar más de 2 cargos, no muestra todos los usuarios

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#227] [4. Bug Reports] [answered]
Hola

Me di cuenta que al usar un selector múltiple / usuarios / filtros con cargos: si uno selecciona más de 1 cargo, no muestra el listado de usuarios. lo comento porque tengo una ca de 40 usuarios aprox y quiero que aparezcan todos en el selector, marcando los cargos que tienen cada uno, pero me muestra 10.

Adjunto code de la question
```
{ 
    "_id" : ObjectId("60b8db4b720e2700094b49bc"), 
    "command" : {
        "commands" : [

        ], 
        "resetIdentifiers" : [

        ], 
        "isCommanded" : null, 
        "values" : [

        ]
    }, 
    "display" : [

    ], 
    "code" : [
        "{\"allow\":\"jobTitle\",\"value\":[\"abogado\",\"administrativo\",\"procurador\"],\"filter\":\"*\"}"
    ], 
    "isActive" : true, 
    "isSystemModel" : false, 
    "contentType" : "application/vnd.cotalker.survey+person", 
    "help" : null, 
    "identifier" : "seleccione_usuario_a_invitar", 
    "isReadOnly" : false, 
    "max" : NumberInt(1), 
    "min" : NumberInt(1), 
    "required" : false, 
    "subtype" : null, 
    "company" : ObjectId("5fb272302f5af2000898d992"), 
    "modifiedAt" : ISODate("2021-06-03T13:54:22.033+0000"), 
    "__v" : NumberInt(1), 
    "exec" : {

    }
}
```
Una solución que tengo, por el momento, es cambiar el selector a uno de colección,  y agrego como propiedad al usuario.

Gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG La lista de usuarios posible está en base a los contactos visibles para el usuario.

Realicé una prueba en testco en dev, y opera correctamente el filtro de jobTitle.

**Reply by NicolasPLG** [reply]
Encontré cuál fue el problema y era que el jobTitle estaba con la letra inicial mayúscula, mientras que companies estaba minúscula (por usar el importador 1.0 del admin antiguo).
Muchas gracias por la respuesta.
